README.md

```diff
@@ -1 +1,170 @@
+r022-0fd54b16-2f03-4f8c-8045-38f9781e9071   ibm-debian-10-8-minimal-amd64-1                    available    amd64   debian-10-amd64                      10.x Buster/Stable - Minimal Install      1               public       provider     none         -   
+r022-54098fbc-4285-4fc3-aa20-d9da85781fa5   ibm-debian-9-13-minimal-amd64-4                    available    amd64   debian-9-amd64                       9.x Stretch/Stable - Minimal Install      1               public       provider     none         -   
+r022-f5387730-7a4b-4f71-9a85-13b05b137953   ibm-redhat-7-6-amd64-sap-applications-1            available    amd64   red-7-amd64-sap-applications         7.x for Applications                      2               public       provider     none         -   
+r022-60d279a0-b328-40eb-a379-595ca53bee18   ibm-redhat-7-6-amd64-sap-hana-1                    available    amd64   red-7-amd64-sap-hana                 7.6 for SAP HANA                          2               public       provider     none         -   
+r022-71ecd746-b847-4fc4-8144-1ad5ca7095fc   ibm-redhat-7-9-minimal-amd64-3                     available    amd64   red-7-amd64                          7.x - Minimal Install                     1               public       provider     none         -   
+```
+
+Default confing file `vars.yml`
+
+    name_prefix: 'perf-scale-test'  #Used as prefix when creating various resources
+    vsi_profile: 'bx2-4x16'   #type of instance, balanced 2 vCPUs and 16 GiB Memory
+    image_id: 'r022-939bbfe2-d823-4c01-98a3-a6f8193731d8' #OS image ID
+    ssh_public_key: <your ssh public key> #cat ~/.ssh/id_rsa.pub
+    ic_instance: 3 #Number of required IBM cloud instances
+    list_of_ports: [22, 80, 443] #open port to access
+
+Run the `ansible-playbook` command with following `EXTRA` cmdline arguments
+
+    ansible-playbook create.yml -e ic_instance=2 \
+                                -e name_prefix='perf-scale-test' \
+                                -e vsi_profile='bx2-4x16' \
+                                -e image_id='r014-02843c52-e12b-4f72-a631-931b4bf6589d' \
+                                -e install_tower=False
+```
+TASK [Print IBM Cloud Instance Floating IPs] ***************************************************************************************************************************************
+ok: [localhost] => {
+    "msg": [
+        "IC instance Floating IP: ",
+        [
+            "169.63.178.143",
+            "169.59.164.19"
+        ]
+    ]
+}
+```
+```
+[cmusali@cmusali ibm_cloud_tower_deploy]$ cat > ic_instance_inventory.ini
+[ic_servers]
+ic1 ansible_host=169.63.178.143
+ic2 ansible_host=169.59.164.19
+
+
+[ic_servers:vars]
+ansible_user = 'root'          
+ansible_ssh_private_key_file=conf/towerperf_id_rsa
```

Comment:
Should we mention that this private key is part of towerperf repo? Also may be we should add a section wherein we tell how to change the ssh key. Basically you need to either pass it via extra vars or change in it vars.yml ?

Reply:
>Should we mention that this private key is part of towerperf repo?

It is an attempt to show what can be added to the inventory. 

>Also may be we should add a section wherein we tell how to change the ssh key. Basically you need to either pass it via extra vars or change in it vars.yml ?

Line 113-120 explains `vars.yaml` default config values.

Reply:
Perfect. Looks good then.